mergin/test/test_client.py

```diff
@@ -573,14 +600,38 @@ def test_clone_project(mc):
     assert any(p for p in projects if p["name"] == test_project and p["namespace"] == API_USER)
 
     cloned_project_name = test_project + "_cloned"
+    test_cloned_project_fullname = API_USER + "/" + cloned_project_name
+
     # cleanup cloned project
     cloned_project_dir = os.path.join(TMP_DIR, cloned_project_name)
     cleanup(mc, API_USER + "/" + cloned_project_name, [cloned_project_dir])
 
-    # clone project
-    mc.clone_project(test_project_fullname, cloned_project_name, API_USER)
+    # clone specifying cloned_project_namespace, does clone but raises deprecation warning
+    with pytest.deprecated_call(match=r"The usage of `cloned_project_namespace` parameter in `clone_project\(\)`"):
+        mc.clone_project(test_project_fullname, cloned_project_name, API_USER)
```

Comment:
I think this should be migrated to `mc.clone_project(test_project_fullname, API_USER + "/" + cloned_project_name)`

also for any other existing clone_project / create_project calls, we should ensure we're using the new API variants

Reply:
Actually the proper call, which you are refering to, is down on line 631. These lines that you highlight are there to test that the backward compatability is not broken and the deprecated message is properly emmited.